changelogs/fragments/81504-update-logical-operation.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - collection dependency resolver - previous code had the same operand on both sides of a logical expression. Fix now updates the operand in the right side of the logical expression (https://github.com/ansible/ansible/pull/81511).
```

Comment:
This is no longer accurate.

Reply:
@munahaf Could you please update the changelog? Thanks

Reply:
Modified the change log.

Reply:
If it doesn't look like a valid changelog, can you please suggest one?